docs/CLI.md

```diff
@@ -487,15 +487,19 @@ Alias: `-v`. Print the version and exit.
 
 Watch files for changes and rerun tests related to changed files. If you want to re-run all tests when a file has changed, use the `--watchAll` option instead.
 
+:::tip
+
+Use `--no-watch` to explicitly disable the watch mode if it was enable using `--watch`. In most CI environments, this is automatically handled for you.
+
+:::
+
 ### `--watchAll`
 
 Watch files for changes and rerun all tests when something changes. If you want to re-run only the tests that depend on the changed files, use the `--watch` option.
 
-Use `--watchAll=false` to explicitly disable the watch mode.
-
 :::tip
 
-In most CI environments, this is automatically handled for you.
+Use `--no-watchAll` to explicitly disable the watch mode if it was enable using `--watchAll`. In most CI environments, this is automatically handled for you.
```

Comment:
```suggestion
Use `--no-watchAll` (or `--watchAll=false`) to explicitly disable the watch mode if it was enabled using `--watchAll`. In most CI environments, this is automatically handled for you.
```

`--no-watch-all` and `--watch-all=false` would also work, but we probably shouldn't overload